src/react/Link/Link.test-d.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-extraneous-dependencies  */
```

Comment:
Just need to update the `devDependencies` patterns in `.eslintrc.js` to fix this.